test/jdk/sun/security/pkcs11/Cipher/TestSymmCiphers.java

```diff
@@ -134,13 +106,12 @@ public void main(Provider p) throws Exception {
     }
 
     private static void test(Cipher cipher, int mode, SecretKey key,
-            AlgorithmParameters params, int firstBlkSize,
-            byte[] in, byte[] answer) throws Exception {
+                             AlgorithmParameters params, int firstBlkSize,
+                             byte[] in, byte[] answer) throws Exception {
         // test setup
         long startTime, endTime;
         cipher.init(mode, key, params);
         int outLen = cipher.getOutputSize(in.length);
-        //debugOut("Estimated output size = " + outLen + "\n");
```

Comment:
I usually do not want to comment out code. Maybe when this test fails one day, the author meant to uncomment them to debug the issue.